ui/src/App.tsx

```diff
@@ -150,7 +166,10 @@ const App = () => {
           <TextField style={{margin:"10px"}} id="outlined-basic" label="Armspan" variant="outlined" />
           <Divider />
           <Typography style={{margin:"10px"}}>2. Upload the route image</Typography>
-          <TextField style={{margin:"10px"}} id="outlined-basic" label="File Upload" variant="outlined" />
+          <Button style={{margin:"10px"}} variant="contained" color="secondary" onClick={() => { handleOpenImage()}}> Upload File <input
+    type="file"
+    style={{ display: "none" }}
+  /></Button>
```

Comment:
This needs to be reformatted. 

1. Line is too long so move "Upload File" to next line.
2. We shouldn't start a new tag a the end of a line and not end the first tag there, it is misleading. For this we can just have input on the next line and probably have all of input on one line.
3. Move the "/>" on the last line up so the end carrot is not floating

Reply:
Fixed in latest commit